claimstore/core/decorators.py

```diff
@@ -1,38 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Following up the removal of `decorators.py`, please also update `docs/api.py`, otherwise one gets:

```
/code/docs/api.rst:57: WARNING: autodoc: failed to import module 'claimstore.core.decorators'; the following exception was raised:
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/site-packages/sphinx/ext/autodoc.py", line 385, in import_object
    __import__(self.modname)
ImportError: bad magic number in 'claimstore.core.decorators': b'\x03\xf3\r\n'
```